src/controllers/commands_controller.js

```diff
@@ -40,6 +43,42 @@ const Things = require('../models/things');
 
 const CommandsController = PromiseRouter();
 
+function internalError(response) {
+  response.status(400).json({
+    message: 'Sorry, something went wrong.',
+  });
+}
+
+function thingNotFound(response, name) {
+  response.status(400).json({
+    message: `Sorry, the thing ${name} wasn't found.`,
+  });
+}
+
+function invalidForDevice(response) {
+  response.status(400).json({
+    message: 'Sorry, I\'m afraid I can\'t do that.',
+  });
+}
+
+function propertyNotFound(response, thing, property) {
+  response.status(400).json({
+    message: `Sorry, the thing ${thing} has no property ${property}.`,
```

Comment:
A bit awkward again. Maybe "Sorry, that thing has no property named ${property}." would make more sense.